features/importers/myhelsinki_places/importer.py

```diff
@@ -61,6 +61,14 @@ class MyHelsinkiImporter(FeatureImporterBase):
 
     source_system = "myhelsinki"
     source_type = "place"
+
+    # Query parameters: http://open-api.myhelsinki.fi/doc#/v1places/listAll
+    api_calls = [
```

Comment:
Optional: could these be moved to a separate configuration file? Under the `myhelsinki_places` folder perhaps. Would possibly make usage of the importer more clear.

Reply:
This might make sense. Though I think I could do a separate PR for that.

Reply:
Separate PR for this is fine as it could also include the unrelated change for `tag_config`.